hw4/hw4.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+if __name__ == "__main__":
+    arr0 = np.zeros(10)
+    arr1000_7 = np.array([i for i in range(1000, 0, -7)])
+    arr_rand = np.random.rand(10)
+
+
+def multiplication_check(list_of_matrix):
+    a = np.shape(list_of_matrix[0])[0]
+    b = np.shape(list_of_matrix[0])[1]

```

Comment:
Рабочий вариант, но для этого обычно используют метод эррея `.shape`, это чуть легче читается
```suggestion
    a = list_of_matrix[0].shape[0]
    b = list_of_matrix[0].shape[1]

```